ai-platform-unified/notebooks/official/custom-training-batch-prediction.ipynb

```diff
@@ -0,0 +1,1213 @@
+{
```

Comment:
<p>Typo: "To make a batch prediction <strong>request</strong>" (Emphasis here for clarity, of course don't bold that word in the notebook)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>Ha we need that spellcheck integration it seems.</p>

Reply:
<p>Fixed</p>